bin/cylc-get-host-metrics

```diff
@@ -35,19 +35,18 @@ keys as requested via the OPTIONS:
 If no options are specified, --load and --memory are invoked by default.
 """
 
-import sys
-if '--use-ssh' in sys.argv[1:]:
-    sys.argv.remove('--use-ssh')
-    from cylc.remote import remrun
-    if remrun():
-        sys.exit(0)
-
 import os
 import re
 from subprocess import Popen, PIPE, CalledProcessError
 import json
 
 from cylc.option_parsers import OptionParser
+import sys
+if '--use-ssh' in sys.argv[1:]:
+    sys.argv.remove('--use-ssh')
+    from cylc.remote import remrun
+    if remrun():
+        sys.exit(0)
 
```

Comment:
@MartinRyan - can you explain why you made this change?  Import order is a style issue, not security, isn't it?  And the same ordering is in many other files in bin/ that you haven't changed. The existing order is just a minor efficiency thing - no point in importing stuff that will never be used if the command just re-invokes itself on a remote host (which is what `remrun` does).

Reply:
codacy was compalining and failing failing the build hilary

Reply:
Fair enough!  Codacy "fails" if the number of flagged issues goes up, but there's a whole bunch we need we need to deal with (potentially via the Codacy equivalent of a `#nosec` - as in this case, I believe - as these static analysis tools blindly apply rules with no understanding of context or reasonable exceptions).